verification/verify_leaderboard_v2.py

```
@@ -0,0 +1,162 @@
+            try:
+                await page.wait_for_selector("video", timeout=3000)
+                play_button = page.locator("button:has-text('Start Tour')")
+                if await play_button.is_visible(timeout=2000):
+                    await play_button.click()
+                    await page.wait_for_timeout(500)
+                skip_button = page.locator("button:has-text('Skip Intro')")
+                if await skip_button.is_visible(timeout=5000):
+                    await skip_button.click()
+                else:
+                    await page.wait_for_selector("video", state="detached", timeout=30000)
+            except:
```

Ruff (0.15.2)

[error] Do not use bare `except`

(E722)